ibtk/include/ibtk/FEDataManager.h

```diff
@@ -118,6 +118,15 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
         {
         }
 
+        /*!
+         * Clear the contents of the cache.
+         */
+        inline void
+        clear()
+        {
+            d_dof_cache.clear();
```

Comment:
I assume that the `DofMap` object itself does not change, but I wonder if it would nonetheless be "safer" to wipe out the `SystemDofMapCache` objects instead, and to repopulate them from scratch as needed. `LDataManager` is already set up to automatically rebuild the cache object if the stored pointer is null.

Reply:
That's a better idea: that way we are covered in case the `DofMap` object is destroyed.

Reply:
Fixed!